JSTests/microbenchmarks/proxy-get.js

```diff
@@ -1,8 +1,10 @@
+(function() {
 var proxy = new Proxy({}, {
     get(target, propertyName, receiver) {
         return 42;
     }
 });
 
-for (var i = 0; i < 1e7; ++i)
```

Comment:
I will revert this change before landing, we can definitely sample `1e7` times given another patch removes `@handleProxyGetTrapResult` and makes `[[Get]]` trap much faster.